.github/s2n_bsd.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#  http://aws.amazon.com/apache2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+#
+set -eu
+export CTEST_OUTPUT_ON_FAILURE=1
+
+if [ "$(uname)" = 'FreeBSD' ]; then
+    export CTEST_PARALLEL_LEVEL=$(sysctl hw.ncpu | awk '{print $2}')
+else
+    export CTEST_PARALLEL_LEVEL=$(sysctl -n hw.ncpuonline)
```

Comment:
Is this really necessary? Why can't they both just use hw.ncpu? It seems like a silly thing to branch on xD Minimally we probably need a comment here.

Reply:
I was just going off of what was there. I'm assuming the script was duplicated for this reason, as there were no other differences.

Reply:
I wouldn't rule out the script being duplicated for no real reason at all :)

Reply:
https://serverfault.com/questions/203207/how-do-i-count-the-number-of-processors-on-an-openbsd-system